docs/pages.jl

```diff
@@ -1,7 +1,6 @@
 # Put in a separate page so it can be used by SciMLDocs.jl
 
-pages = [
-    "Home" => "index.md",
```

Comment:
This change makes the sidebar pretty ugly since it will use `LinearSolve.jl: High-Performance Unified Linear Solvers` (the title on the `index.md` page) instead of `Home`. (Compare https://linearsolve.sciml.ai/dev/ with https://linearsolve.sciml.ai/stable/)

Reply:
Yes. For now I think having the results show up properly in Google is worth more than a cleaner sidebar. When we can unlink the two then it will get updated.